### DISCUSSION TITLE
Section dashboard: conditional cards in horizontal row

### DISCUSSION THREAD

**Discussion by huperaisan** [home-assistant/frontend#21396] [Dashboard]
I have recreated my Masonry dashboard in the new Sections format, and noticed a difference in behaviour. I have a horizontal row of conditional tile cards for door contact sensors that are only displayed if the door is open. In the Masonry dashboard, if none of the conditional cards are visible, the entire row would disappear and not take up space, with the next element vertically moving up to fill in. In the Sections style, the same situation results in all the conditional cards being (correctly) invisible, but a blank row remains.

Row being edited:
<img width="305" alt="image" src="https://github.com/user-attachments/assets/9191ad79-2d3e-4dc3-81fa-b9df92b1987e">

Blank row that remains when all cards are invisible:
<img width="305" alt="image" src="https://github.com/user-attachments/assets/9751232f-d079-4d6a-a860-2a9792e7c1aa">

**Comment by potelux** [comment]
I am not able to reproduce this issue.  It is possible that this has already been resolved since you posted this back in July.  Can you please check again and let us know if the issue persists?

**Reply by huperaisan** [reply]
Hello! Apologies for late reply. I've upgraded to the latest HA version and the issue is still _kind of_ there. The space is much smaller than before:

<img width="325" alt="image" src="https://github.com/user-attachments/assets/36e9ffe9-b407-4df7-8288-36470e12d8c4">

I've included a bit more in the screenshot, so you can see the vertical gap under the "garage door" box (which is where the hidden tiles are) is slightly larger than under the "to-do" box. It's a pretty minor issue now, and probably not a priority.
